src/main/java/com/google/error_analyzer/backend/Pagination.java

```diff
@@ -0,0 +1,135 @@
+/**Copyright 2019 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+package com.google.error_analyzer.backend;
+
+import com.google.common.collect.ImmutableList;
+import com.google.error_analyzer.data.ErrorFixes;
+import com.google.error_analyzer.data.SearchErrors;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.lang.*;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+import org.elasticsearch.search.SearchHit;
+import org.elasticsearch.search.SearchHits;
+
+
+@WebServlet("/pagination")
+public class Pagination extends HttpServlet {
```

Comment:
Are we adding "Servlet" as suffix to such classes?
Here it would be "PaginationServlet".
Either way, please check with others and keep it homogeneous.
I am okay either way.